website/content/en/docs/building-operators/golang/advanced-topics.md

```diff
@@ -113,10 +113,196 @@ func init() {
 * After adding new import paths to your operator project, run `go mod vendor` if a `vendor/` directory is present in the root of your project directory to fulfill these dependencies.
 * Your 3rd party resource needs to be added before add the controller in `"Setup all Controllers"`.
 
-### Metrics
+### Monitoring and Observability
+This section aims to explain how to expose the first operator's custom metrics, [alerts] and [recording rules]. It focuses on the technical aspects, and demonstrates the implementation over the sample [memcached-operator]. 
 
-To learn about how metrics work in the Operator SDK read the [metrics section][metrics_doc] of the Kubebuilder documentation.
+For more information regarding operators' monitoring, please follow [observability-best-practices].  
 
+#### Prerequisites
+The following steps are required in order to inspect the operator's custom metrics, alerts and recording rules:
+- Deploy [kube-prometheus] on your cluster. This will set a Prometheus instance on the cluster, with which we will inspect the metrics, alerts and recording rules.
```

Comment:
Do we specifically need "kube-prometheus"? Can we just say a "prometheus instance" as a pre-req? Users could use prometheus-operator or even deploy a prometheus instance directly.

Reply:
We could add that using cube-prometheus is recommended. Something as in here (https://book.kubebuilder.io/reference/metrics.html#exporting-metrics-for-prometheus)

Reply:
Done, thanks